core/unsigneddata.go

```diff
@@ -82,7 +82,12 @@ type attestationDataJSON struct {
 	Duty *eth2v1.AttesterDuty    `json:"attestation_duty"`
 }
 
-// AggregatedAttestation wraps Attestation and implements the UnsignedData interface.
+// NewAggregatedAttestation returns a new aggregated attestation.
+func NewAggregatedAttestation(att *eth2p0.Attestation) AggregatedAttestation {
+	return AggregatedAttestation{Attestation: *att}
+}
+
+// AggregatedAttestation wraps un unsigned aggregated attestation and implements the UnsignedData interface.
```

Comment:
add a test for this in unsigneddata_test and maybe convert existing ones into table tests